elliottlib/cli/change_state_cli.py

```diff
@@ -56,53 +59,53 @@ def change_state_cli(runtime, state, advisory, default_advisory_type, noop):
 
     $ elliott change-state -s NEW_FILES -a 123456 --noop
 """
-    if not (bool(advisory) ^ bool(default_advisory_type)):
-        raise click.BadParameter("Use only one of --use-default-advisory or --advisory")
+    count_flags = sum(map(bool, [advisory, default_advisory_type, default_advisories]))
+    if count_flags > 1:
+        raise click.BadParameter("Use only one of --use-default-advisory or --advisory or --default-advisories")
 
-    runtime.initialize(no_group=default_advisory_type is None)
+    runtime.initialize(no_group=not (default_advisory_type or default_advisories))
 
+    advisories = []
     if default_advisory_type is not None:
         advisory = find_default_advisory(runtime, default_advisory_type)
 
-    if noop:
-        prefix = "[NOOP] "
-    else:
-        prefix = ""
-
-    try:
-        e = Erratum(errata_id=advisory)
-
-        if e.errata_state == state:
-            green_prefix("{}No change to make: ".format(prefix))
-            click.echo("Target state is same as current state")
-            return
-        # we have 5 different states we can only change the state if it's in NEW_FILES or QE
-        # "NEW_FILES",
-        # "QE",
-        # "REL_PREP",
-        # "PUSH_READY",
-        # "IN_PUSH"
-        if e.errata_state != 'NEW_FILES' and e.errata_state != 'QE':
-            if default_advisory_type is not None:
-                raise ElliottFatalError("Error: Could not change '{state}' advisory {advs}, group.yml is probably pointing at old one".format(state=e.errata_state, advs=advisory))
+    if advisory:
+        advisories.append(advisory)
+
+    if default_advisories:
+        for impetus in ["rpm", "image", "extras"]:
+            advisories.append(runtime.group_config.advisories[impetus])
```

Comment:
Please don't hardcode impetuses here.

```suggestion
        advisories = [runtime.group_config.advisories.values()]
```

Reply:
Thank you, I'll remember this!